src/brain/gocdSlackFeeds/deployFeed.ts

```diff
@@ -1,3 +1,5 @@
+import '@sentry/tracing';
```

Comment:
Suggested by @lforst 
https://github.com/getsentry/sentry-javascript/issues/4731#issuecomment-1126157027

Reply:
Hi, that comment is a bit outdated. We deprecated the Sentry tracing package (it's just no-ops now) so we could remove it entirely.